src/sb_sw_lib_tests.c.h

```diff
@@ -1620,11 +1620,105 @@ _Bool sb_test_sw_invalid_scalar(void)
     return 1;
 }
 
+// Test that sweet-b is not vulnerable to the "Psychic Paper" attack.
+// More specifically, verify that for some signature (r, s) if r or s = 0
+// signature verification returns invalid.
+static _Bool sb_test_invalid_sig(const sb_byte_t invalid[static const 1],
```

Comment:
```suggestion
static _Bool sb_test_invalid_sig(const sb_byte_t invalid[static const SB_ELEM_BYTES],
```

You're using more than one byte of this array...